test/core/integration/concrete/lockup-tranched/get-tranches/getTranches.tree

```diff
@@ -1,4 +1,4 @@
-GetTranches_LockupTranched_Integration_Concrete_Test
+GetTranches_Integration_Concrete_Test
```

Comment:
I mentioned in the descriotion: `Write missing tests, check coverage` which is an unchecked task. It was a part of that. I wanted to get your review first because restructuring is a bigger task than this.